File: app/src/main/java/com/simplemobiletools/calculator/operation/PlusOperation.java
package com.simplemobiletools.calculator.operation;

import com.simplemobiletools.calculator.operation.base.BinaryOperation;
import com.simplemobiletools.calculator.operation.base.Operation;

public class PlusOperation extends BinaryOperation implements Operation {

    protected PlusOperation(double baseValue, double secondValue) {
        super(baseValue, secondValue);
    }

    @Override
    public double getResult() {
        return baseValue + secondValue;
    }
}

File: app/src/main/kotlin/com/simplemobiletools/calculator/operation/PlusOperation.kt
package com.simplemobiletools.calculator.operation

import com.simplemobiletools.calculator.operation.base.BinaryOperation
import com.simplemobiletools.calculator.operation.base.Operation

class PlusOperation(baseValue: Double, secondValue: Double) : BinaryOperation(baseValue, secondValue), Operation {

    override fun getResult() = baseValue + secondValue
}
